src/inspector_agent.cc

```diff
@@ -706,6 +791,77 @@ void Url(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(OneByteString(env->isolate(), url.c_str()));
 }
 
+static void InvokeAsyncTaskFnWithId(void (Agent::*asyncTaskFn)(void*),
+                                    const FunctionCallbackInfo<Value>& args);
+
+static void* GetAsyncTask(int64_t asyncId);
```

Comment:
I personally prefer helper functions to be defined below the place where they are used, it makes the code easier to read (to me). Having said that, I am happy to follow whatever convention Node.js uses. I'll apply your suggestion.